common/djangoapps/student/views.py

```diff
@@ -2552,16 +2565,24 @@ def password_reset_confirm_wrapper(request, uidb36=None, token=None):
 
     if request.method == 'POST':
         password = request.POST['new_password1']
-        is_password_valid, password_err_msg = validate_password(user, password)
-        if not is_password_valid:
+        valid_link = False
+        error_message = validate_password_security_policy(user, password)
+        # if security is violated, no need to validate password
+        if not error_message:
+            error_message = validate_password(password)
```

Comment:
Add a comment here saying we should validate password if no security policy is violated